providers/tentacle.rb

```diff
@@ -134,6 +136,8 @@
   # Make sure enabled and started
   service = windows_service service_name do
     action [:enable, :start]
```

Comment:
Last thing. This should include the following so that failures do not end up in logs

```
sensitive true if service_user
```